flink-table/flink-table-runtime-blink/src/test/java/org/apache/flink/table/runtime/rank/BaseTopNFunctionTest.java

```diff
@@ -41,14 +44,10 @@
 import java.util.ArrayList;
 import java.util.List;
 
-import static org.apache.flink.table.runtime.util.StreamRecordUtils.deleteRecord;
-import static org.apache.flink.table.runtime.util.StreamRecordUtils.record;
-import static org.apache.flink.table.runtime.util.StreamRecordUtils.retractRecord;
-
 /**
- * Base Tests for all subclass of {@link AbstractRankFunction}.
+ * Base Tests for all subclass of {@link AbstractTopNFunction}.
  */
-abstract class BaseRankFunctionTest {
+abstract class BaseTopNFunctionTest {
```

Comment:
I would suggest to use `TopNFunctionTestBase` as class name. Because a `*Test` suffix will be recognized as JUnit tests.